crates/bevy_dev_tools/src/fps_overlay.rs

```diff
@@ -86,13 +170,75 @@ fn update_text(diagnostic: Res<DiagnosticsStore>, mut query: Query<&mut Text, Wi
     }
 }
 
+fn update_frame_time_values(
+    mut frame_time_graph_materials: ResMut<Assets<FrametimeGraphMaterial>>,
+    diagnostics_store: Res<DiagnosticsStore>,
+) {
+    let Some(frame_time) = diagnostics_store.get(&FrameTimeDiagnosticsPlugin::FRAME_TIME) else {
+        return;
+    };
+    let frame_times = frame_time
+        .values()
+        // convert to millis
+        .map(|x| *x as f32 / 1000.0)
+        .collect::<Vec<_>>();
+    for (_, material) in frame_time_graph_materials.iter_mut() {
+        material.values = frame_times.clone();
+    }
+}
+
 fn customize_text(
```

Comment:
Nit: Probably should rename this system to something like `customize_display` now that it includes a graph, not just text.

Reply:
done